.github/workflows/ci.yml

```diff
@@ -26,17 +26,11 @@ jobs:
 
   Lint:
     runs-on: ubuntu-latest
-    env:
-      # not needed for linting
-      PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: '1'
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: '20.x'
-          cache: 'npm'
-      - run: npm ci --legacy-peer-deps
-      - run: npm run lint
+      - uses: biomejs/setup-biome@v2 # detects version from package.json
+        run: biome ci
```

Comment:
This is basically instant now in CI, since it doesn't require `npm install-ing` the project to lint. This will make identifying style errors _much_ faster.